configs/train/aaf/double_poisson_aaf_cfg.yaml

```diff
@@ -0,0 +1,27 @@
+experiment_name: aaf
+random_seed: 1998
+head_type: double_poisson
+dataset:
+  type: image
+  spec: aaf
+training:
+  accelerator: gpu
+  batch_size: 32
+  chkp_dir: chkp
+  chkp_freq: 6
```

Comment:
Does this mean you saved a model every 6 epochs for 200 epochs? We can probably delete a lot of those intermediate ones haha

Reply:
Looking at `ava`'s storage, it's at 98% capacity now. My guess is you have a ton of auxiliary models taking up storage space now. Delete everything you don't need, then run `df -h` and see if you can get the number down for `/dev/nvme0n1p2`

Reply:
What would you like the frequency to be? I think we just copied the coco config that you sent us.